src/checkoutservice/middleware.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Google LLC
```

Comment:
Do we want this or something like "Copyright The OpenTelemetry Authors"

Reply:
Anything net new should be credited to otel!